app/views/components/_glance-metric.html.erb

```diff
@@ -0,0 +1,43 @@
+<%
+  name ||= false
+  figure ||= false
+  period ||= false
+  link_text ||= false
+  link_url ||= false
+  measurement_display_label ||= ""
+  measurement_explicit_label ||= ""
+  context ||= ""
+  trend_percentage ||= ""
+%>
+<% if name && figure && period %>
+  <div class="app-c-glance-metric govuk-body">
+    <h3 class="app-c-glance-metric__heading govuk-heading-s"><%= name %></h3>
+    <span class="app-c-glance-metric__figure govuk-!-font-weight-bold govuk-!-font-size-36 "><%= figure %><span class="app-c-glance-metric__measurement" <% if measurement_explicit_label %>aria-label="<%= measurement_explicit_label %>"<% end %>><%= measurement_display_label %></span></span>
```

Comment:
I would avoid using overwrites here.
We can define the font styling in `__figure` using a mixin: `govuk-font($size: 36, $weight: bold)`